spec/factories.rb

```diff
@@ -0,0 +1,9 @@
+FactoryBot.define do
```

Comment:
I think this file should be called `users.rb` and located in the folder `spec/factories`. Then when you have a new model you would have a new factory file like `new_models.rb` in the same folder.